player.gd

```diff
@@ -0,0 +1,77 @@
+extends CharacterBody3D
```

Comment:
Player script looks good! I would just add more comments to specify what you mean by "neck"

Reply:
I agree, it looks good! Maybe some comments explaining some simple comments about the lines regarding the rotation of the camera would be nice.